From: Modestas Vainius <modax@debian.org>
Subject: Relax kde-workspace dependency on kdelibs
Origin: vendor
Forwarded: not-needed

Otherwise we can't build with 4.8.3. Moreoever, such a strict dependency is
pretty pointless anyway.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,7 +12,7 @@ set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SO
 
 #search packages used by KDE
 set(QT_MIN_VERSION "4.7.0")
-set(KDE_MIN_VERSION "4.8.4")
+set(KDE_MIN_VERSION "4.8.0")
 set(SOPRANO_MIN_VERSION "2.5.63")
 
 OPTION(WITH_XINERAMA "Xinerama support for multi-headed X displays" ON)
